### ISSUE TITLE
Model Query

### ISSUE THREAD

**Issue by DivyaWadehra** [facebookresearch/UnsupervisedQA#12]
How can we create our own data specific model..?

**Comment by patrick-s-h-lewis** [comment]
Hi Divya, Can you give more information?

**Comment by monk1337** [comment]
@patrick-s-h-lewis  I am facing the same issue, I have two questions : 

1) How to fine-tune the pre-trained model for the domain-specific task instead of training it from scratch.
2) How to train from scratch if I have the only corpus no ground truth?

Thank you for awesome work :)

**Comment by patrick-s-h-lewis** [comment]
Hi!

Are you trying to build Question Answering models for Question Generation models?
What is your end goal? 

That'll help me understand what you need to achieve and then I can give you better advice :)

Patrick

**Comment by monk1337** [comment]
@patrick-s-h-lewis  Thank you for the quick reply.
Yes, I am working on the Question Generation task.

**Comment by askaydevs** [comment]
@patrick-s-h-lewis Might it be possible to train a transformer decoder model like gpt-2 for question generation ?

**Comment by patrick-s-h-lewis** [comment]
@monk1337 - so you want to generate questions that are relevant to your particular corpus of texts? I'd suggest looking at supervised question generation, the results will probably be much better. This is a research codebase for unsupervised question answering, which doesnt fit your needs as well.

@askaydevs Sure, this is done a lot. But this codebase is for unsupervised question generation, which has different challenges.

**Comment by monk1337** [comment]
@patrick-s-h-lewis Sure, 

1) But I have very big corpus without any questions ( labels ), How can I go for supervised question generation?

2) I can use pre-trained supervised question generation for my corpus that is one option, can you suggest pre-trained models for supervised question generation?

**Comment by patrick-s-h-lewis** [comment]
> But I have very big corpus without any questions ( labels ), How can I go for supervised question generation?

Train a supervised question generator on other data and then use it to generate questions on your corpus

> I can use pre-trained supervised question generation for my corpus that is one option, can you suggest pre-trained models for supervised question generation?

there are many options. E.g. https://arxiv.org/abs/1905.03197, or T5 or BART

As this is moving away from the purpose of this repository, this issue is now closed.
